MNIST_skeleton.py

```diff
@@ -15,11 +15,14 @@
                    ])),
     batch_size=64, shuffle=True)
 
-
-# TODO: implement squash function
 def squash(input):
-    return input
-
+    """
+    Squashing function for a tensor.
+    :param input: torch.Tensor
+    """
+    norm = input.norm()
```

Comment:
The `norm()` method is not implemented for all tensor types. It works with the default tensor, but it will crash for a `LongTensor`. 

I think we should be good, but it's still an important limitation to keep in mind.